src/main/io/serial_4way.c

```diff
@@ -422,6 +422,11 @@ void esc4wayProcess(serialPort_t *mspPort)
     uint8_t *InBuff;
     ioMem_t ioMem;
 
+    if (!mspPort)
+    {
```

Comment:
Styling hint: K&R style has braces on a new line for function definitions, but on the same line everywhere else.